torch/csrc/dynamo/compiled_autograd.h

```diff
@@ -356,6 +356,9 @@ class CompiledNodeArgs {
     for (auto& i : fn->post_hooks()) {
       i->compiled_args(*this);
     }
+    for (auto& i : fn->tensor_post_acc_grad_hooks()) {
+      i->compiled_args(*this);
+    }
```

Comment:
Might be clearer to immediately error just like we do for retains_grad_hooks tbh 
The next best thing is to at least add a comment here saying that we don't support this. Otherwise if i'm reading these that wouldn't be immediately obvious to me.

Reply:
Agreed about a comment saying this doesn't work well.

Reply:
I ended up having it error